src/components/feedback.tsx

```diff
@@ -130,7 +130,7 @@ export const FeedbackSection: React.FC<BoxProps> = props => {
         mt={space(['extra-loose', 'extra-loose', 'base-loose'])}
       >
         <Link
-          href={`https://github.com/blockstack/docs/edit/master/src/${editPage}.md`}
+          href={`https://github.com/blockstack/docs/edit/master/src/pages${editPage}.md`}
```

Comment:
This doesn't need a trailing slash? The original looks like `editPage` wouldn't start with a slash.

Reply:
the pathname is always starts with `/`